terragrunt/aws/rds/rds.tf

```diff
@@ -9,7 +9,6 @@ module "rds_cluster" {
   engine         = "aurora-postgresql"
   engine_version = "14.6"
   instances      = var.database_instances_count
-  instance_class = "db.serverless"
```

Comment:
Instead of getting rid of this, could be set the value explicitly just so we know where its value is coming from:
```hcl
instance_class = "db.t3.medium"
```

Reply:
Done!